### PR TITLE
New version: Tutanota.Tutanota version 259.241213.0

### PR DESCRIPTION
**Pull request by raypappa**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199979)

### CODE REVIEW

**Comment by raypappa** [comment]
@microsoft-github-policy-service agree

File: manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+InstallerLocale: en-US
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2024-12-16
+AppsAndFeaturesEntries:
+- DisplayName: "Tuta Mail 259.241213.0\x03肁"
+  Publisher: Tutao GmbH
```
Comment (DuckDuckStudio) [inline manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.installer.yaml:15]:
```suggestion
#AppsAndFeaturesEntries: --> By komac
#- DisplayName: "Tuta Mail 259.241213.0\x03肁"
#  Publisher: Tutao GmbH
```

Maybe we don't need these entries?

<details>
<summary>原文</summary>
也许我们不需要这些条目？

</details>

**Review by DuckDuckStudio — APPROVED** [review]

File: manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  - Forwarding an email with inline images removes images from viewer #8003
+  - It is not possible to close the dropdown without selecting an option with keyboard/screen reader #7999
+  - Next year price shown if it is the same price as the first year #7935
+  - printing email in dark mode results in gray text color #5781
+  Milestone
+  Asset Checksums (SHA256)
+  tutanota-desktop-linux.AppImage:
+  3b99130d3399c47fc7eb239188c1079f9e40a2e39a4ba8ce0123d9fa250599f5tutanota-desktop-win.exe:
+  fc3dbe41aee2e73c7f63cc1d0cce3225169ade6633a938f5b98afb294cd47bc3tutanota-desktop-mac.dmg:
+  1187aa8ceb72a0ceebf0f003defbadcad957d5545d26070229ab4e17d31d9402
```
Comment (DuckDuckStudio) [inline manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.locale.en-US.yaml:51]:
```suggestion

```
These generated by komac were wrong, we can remove that part.

File: manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+PackageVersion: 259.241213.0
+InstallerLocale: en-US
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2024-12-16
+Installers:
+- Architecture: x86
```
Comment (DuckDuckStudio) [inline manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.installer.yaml:14]:
```suggestion
- Architecture: x64
```
This is probably an x64 application.

File: manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  Scope: user
+  InstallerUrl: https://github.com/tutao/tutanota/releases/download/tutanota-desktop-release-259.241213.0/tutanota-desktop-win.exe
+  InstallerSha256: FC3DBE41AEE2E73C7F63CC1D0CCE3225169ADE6633A938F5B98AFB294CD47BC3
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
+  Scope: machine
```
Comment (DuckDuckStudio) [inline manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.installer.yaml:21]:
```suggestion
- Architecture: x64
  Scope: machine
  ElevationRequirement: elevatesSelf
```
When installing to the machine scope, this installer requires administrator privileges.

**Review by DuckDuckStudio — COMMENTED** [review]
Successfully installed silently in a Windows 11 x64 VM using the `/S /ALLUSERS` parameter.
image

**Review by DuckDuckStudio — CHANGES_REQUESTED** [review]
This is probably an x64 application.
I have already proposed changes earlier.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**